usaspending_api/awards/models.py

```diff
@@ -212,14 +212,15 @@ def get_or_create_summary_award(piid=None, fain=None, uri=None, parent_award_id=
 
         raise ValueError(
             'Unable to find or create an award with the provided information: piid={}, fain={}, uri={}, parent_id={}'.format(
-            piid, fain, uri, parent_award_id))
+             piid, fain, uri, parent_award_id))
 
     class Meta:
         db_table = 'awards'
 
 
 class AwardAction(DataSourceTrackedModel):
     award = models.ForeignKey(Award, models.CASCADE, related_name="actions")
+    usaspending_unique_transaction_id = models.CharField(max_length=256, blank=True, null=True)
```

Comment:
Good idea. Is there any value in naming this something more generic (e.g., `source_unique_transaction_id`) and also using it to capture the surrogate key of the applicable broker table when running the submission ETL?